charts/vulnerability-operator/values.yaml

```diff
@@ -6,6 +6,8 @@ image:
   repository: ghcr.io/ckotzbauer/vulnerability-operator
   tag: ""
   pullPolicy: IfNotPresent
+  pullSecrets:
+    - exampleSecret
```

Comment:
Please use an empty array `pullSecrets: []` as default value, as `exampleSecret` is not available in any cluster.